x/government/client/cli/query.go

```diff
@@ -45,7 +45,7 @@ func getCmdGetGovernmentAddrFunc(cmd *cobra.Command, args []string, cdc *codec.C
 	route := fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryGovernmentAddress)
 	res, _, err := cliCtx.QueryWithData(route, nil)
 	if err != nil {
-		fmt.Printf("Could not get government address: \n %s", err)
+		return fmt.Errorf("could not get government address: \n %s", err)
```

Comment:
```suggestion
		return fmt.Errorf("could not get government address: %s", err)
```

Let's avoid multiline output whenever possible